apps/mymedicare_cb/authorization.py

```diff
@@ -46,25 +46,28 @@ class OAuth2ConfigSLSx(object):
     See the following for more details about the endpoint usage:
     https://confluence.cms.gov/pages/viewpage.action?spaceKey=SLS&title=SLSx%3A+Client+Onboarding+Guide
     """
+    redirect_uri = settings.MEDICARE_SLSX_REDIRECT_URI
+
+    healthcheck_endpoint = settings.SLSX_HEALTH_CHECK_ENDPOINT
+    healthcheck_endpoint_ssl = settings.SLSX_HEALTH_CHECK_ENDPOINT_SSL
+
     token_endpoint = settings.SLSX_TOKEN_ENDPOINT
     token_endpoint_aca_token = settings.MEDICARE_SLSX_AKAMAI_ACA_TOKEN
-    redirect_uri = settings.MEDICARE_SLSX_REDIRECT_URI
+    token_endpoint_ssl = settings.SLSX_TOKEN_ENDPOINT_SSL
+
     signout_endpoint = settings.SLSX_SIGNOUT_ENDPOINT
+    signout_endpoint_ssl = settings.SLSX_SIGNOUT_ENDPOINT_SSL
+
     userinfo_endpoint = settings.SLSX_USERINFO_ENDPOINT
-    healthcheck_endpoint = settings.SLSX_HEALTH_CHECK_ENDPOINT
-    verify_ssl = getattr(settings, 'SLSX_VERIFY_SSL', False)
+    userinfo_endpoint_ssl = settings.SLSX_USERINFO_ENDPOINT_SSL
 
     def __init__(self):
         self.auth_token = None
         self.user_id = None
         self.signout_status_code = None
-        self.signout_status_mesg = None
         self.token_status_code = None
-        self.token_status_mesg = None
         self.userinfo_status_code = None
-        self.userinfo_status_mesg = None
         self.validate_signout_status_code = None
-        self.validate_signout_status_mesg = None
```

Comment:
NOTE: These "_status_mesg" variables ended up not getting used from a previous PR. So removing them here.